addons/popochiu/engine/popochiu.gd

```diff
@@ -152,23 +152,16 @@ func _ready() -> void:
 	am = load(PopochiuResources.AUDIO_MANAGER).instantiate()
 	
 	# Set the Graphic Interface node
-	if settings.graphic_interface:
-		gi = settings.graphic_interface.instantiate()
-		gi.name = 'GraphicInterface'
-	else:
-		gi = load(PopochiuResources.GUI_ADDON_FOLDER).instantiate()
+	gi = load(PopochiuResources.GUI_GAME_SCENE).instantiate()
```

Comment:
Is this ALWAYS available due to the default? What if a game has a custom GUI selected and no GUI is available to instantiate?

Reply:
What do you mean by "a custom GUI selected"?

The GUI scene should be always available because Popochiu creates the corresponding files for the GUI when one closes the Setup popup the first time. But thanks to your comment, I just remembered that I have to include a validation to prompt the user to select a GUI in case they come from a version of the plugin prior to beta 1 (I don't remember in which version we included the GUI templates).